commit 850c646b0b150f9e2efc0f8d2ee93241fd19db79
Author: Enno Boland <g@s01.de>
Date:   Thu Oct 18 19:14:08 2018 +0200

    Revert "skip GTK detection if we're cross-compiling (#588)"
    
    This reverts commit cc3a5705099019e01dc59f7907d0c5b1b85878d8.

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 395848b..39c23b0 100644
--- ./CMakeLists.txt
+++ ./CMakeLists.txt
@@ -39,7 +39,7 @@ endif()
 # Dependencies
 ###
 find_package(LibUSB REQUIRED)
-if (NOT APPLE AND NOT WIN32 AND NOT CMAKE_CROSSCOMPILING)
+if (NOT APPLE AND NOT WIN32)
 	find_package(PkgConfig)
 	pkg_check_modules(gtk gtk+-3.0)
 endif ()
